app/scripts/controllers/builds.js

```diff
@@ -8,7 +8,17 @@
  * Controller of the openshiftConsole
  */
 angular.module('openshiftConsole')
-  .controller('BuildsController', function ($routeParams, $scope, DataService, $filter, LabelFilter, Logger, $location, BuildsService, ProjectsService) {
+  .controller('BuildsController', function (
+    $routeParams,
+    $scope,
+    APIService,
+    DataService,
+    $filter,
+    LabelFilter,
+    Logger,
+    $location,
+    BuildsService,
+    ProjectsService) {
```

Comment:
Let's sort the dependencies to keep the lists manageable

Reply:
sort done